chain/ethereum/tests/manifest.rs

```diff
@@ -71,7 +71,7 @@ async fn resolve_manifest(text: &str) -> SubgraphManifest<graph_chain_ethereum::
     resolver.add(id.as_str(), &text);
     resolver.add("/ipfs/Qmschema", &GQL_SCHEMA);
     resolver.add("/ipfs/Qmabi", &ABI);
-    resolver.add("/ipfs/Qmmapping", &MAPPING);
+    resolver.add("/ipfs/Qmmapping", &MAPPING_WITH_IPFS_FUNC_WASM);
```

Comment:
Is this `.wasm` file a compiled version of this code?

```ts
export function handleGet(call: getCall): void {}
```

Reply:
I just switching it to some valid wasm file, the contents don't matter for this test.